topic/Azure function - event hub triggered/Azure Function/crate_writer.py

```diff
@@ -0,0 +1,127 @@
+import time
+import logging
+from crate import client
+from crate.client.exceptions import ProgrammingError, IntegrityError
+
+
+class CrateWriter:
```

Comment:
```suggestion
class CrateDBWriter:
```
Just a small naming detail for Crate -> CrateDB